interpreter/src/functions.rs

```diff
@@ -436,11 +481,31 @@ pub fn max(Arguments(args): Arguments) -> Result<Value> {
     // If items is a list of values, then operate on the list
```

Comment:
This is much better, I included it in my PR. Also fixed the failing doc comment to use a simpler `sum` example since I figured all the `PartialOrd` stuff sort of distracts from the point the example was making, but if you prefer I can change it back to `max` and fix it

Reply:
Works for me!